data/hai/midas.txt

```diff
@@ -0,0 +1,1010 @@
+		not "Midas: Who is the mad one?: offered"
+		not "Midas: Betrayal For The Bars: offered"
+	on offer
+		"midas: bars" = "outfit: Gold Bars"
+		fail
+
+mission "Midas: Who is the mad one?"
+	outfitter
+	name "A golden opportunity!"
+	description "Go to any pirate world to see if you can sell these gold bars for a good price."
+	to offer
+		has "outfit: Gold Bars"
+	to accept
+		"outfit: Gold Bars" < "midas: bars"
+	on offer
+		conversation
+			`	You had just finished making the transactions for gold bars you sold when you hear someone shouting. "STOP!" You immediately notice the voice is very old. He comes to you in a rush, seemingly alarmed, and addresses the one you sold the bars to. "These are cursed! You should get rid of them. As soon as you can!" Your buyer does not seem particularly surprised by this encounter.`
+			`	The excited man then takes a closer look at the gold bars, even going as far as grabbing the last one you were giving to the buyer to take a closer look. He then stares at you. "How did you even acquire these?" he asks, but he does not give you time to respond. "Di-did you join them? If you did, I am very sorry for any disturbance I may have caused, sir!" he exclaims in fear, moving back and kneeling, as if wanting to apologize, and he presents the bar back as if it was some kind of offering. The buyer energetically takes the bar from his hands, not interested to hear any more, and gets away before you can react. You then turn back to this mysterious stranger.`
+			choice
+				`	"Who are 'they'?"`
+					goto who
+				`	"I got these after a duel."`
+				`	"Where I got these bars is none of your business."`
+					goto wontsay
+				`	(Ignore him.)`
+					decline
+			`	"Ha! I made the same mistake, once, plundering them from a ship. It almost was my last."`
+			label who
+			`	He looks at you, fascinated. "You truly do not know, do you?" He stops for a few seconds, examining your face as if the truth was hidden behind your left ear. "Good! Then you may still be free. Do not make the mistake I did, or this gold will corrupt you! They will come for it! Stash it away, or sell it whilst you still can! It is not worth the fabulous prices they may offer for it!"`
+			`	He continues, finally and painfully catching his breath. "I went to the pirate worlds! To their territory! What a fool I was. Yes, it was a lot of credits, but not for me. I got plundered myself on the way there!"`
+			choice
+				`	"A good price you say? That sounds interesting."`
+				`	"You're right, it is not worth it, I will get rid of them."`
+					goto rid
+				`	"I just sold all of my bars but if they are this interesting I'll be looking for more."`
+					to display
+						not "outfit (all): Gold Bars"
+			`	His eyes open fully in despair, "NO!! You fool! You won't be able to say I did not warn you. You truly are a lost cause, perhaps the bars have already affected you! At least, make sure to have enough escorts, and watch your back.`
```

Comment:
```suggestion
			`	His eyes open fully in despair, "NO!! You fool! You won't be able to say I did not warn you. You truly are a lost cause. Perhaps the bars have already affected you! At any rate, make sure to have enough escorts, and watch your back.`
```
I suppose this could so be thought of as a pun.